crates/holochain/src/conductor/handle.rs

```diff
@@ -281,6 +281,14 @@ pub trait ConductorHandleT: Send + Sync {
     #[cfg(any(test, feature = "test_utils"))]
     async fn add_test_app_interface(&self, id: super::state::AppInterfaceId)
         -> ConductorResult<()>;
+
+    #[cfg(any(test, feature = "test_utils"))]
+    /// Check whether this conductor should skip gossip.
+    fn should_skip_publish(&self) -> bool;
+
+    #[cfg(any(test, feature = "test_utils"))]
+    /// For testing we can choose to skip publish.
+    fn set_skip_publish(&self, skip_publish: bool);
```

Comment:
This is OK, I'm tempted to pull these into a `config.dev` section of `ConductorConfig` though.

Reply:
We might also want to set things like "turn off auto arc resizing"

Reply:
the comment for `should_skip_publish` says "should skip gossip."

Reply:
Maybe we want to defer this?

Reply:
Comment is updated